src/views/Pools/components/PoolsTable/Cells/EarningsCell.tsx

```diff
@@ -67,12 +68,20 @@ const EarningsCell: React.FC<EarningsCellProps> = ({ pool, account, userDataLoad
   hasEarnings = isAutoVault ? hasAutoEarnings : hasEarnings
   earningTokenDollarBalance = isAutoVault ? autoUsdToDisplay : earningTokenDollarBalance
 
+  const timeDiffSinceLastAction = (Date.now() - lastActionInMs) / 1000 / 60 / 60
```

Comment:
We have `date-fns` as a dependency in this repo. You can use [differenceInHours](https://date-fns.org/v2.23.0/docs/differenceInHours) helper from there, like this:

```javascript
import { differenceInHours } from "date-fns";

...

const timeDiffSinceLastAction  = differenceInHours(Date.now(), lastActionInMs)
```